examples/discordjs/src/Commands/ping.js

```diff
@@ -0,0 +1,10 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('ping')
+		.setDescription('Get the bots latency!'),
+    execute: async (interaction, client) => {
```

Comment:
Broken formatting, this should have the same indentation as `data`.

Reply:
It does in my vsc
![image](https://user-images.githubusercontent.com/69105206/139538986-427dbcba-189c-46ca-8530-8c9bd1d210d6.png)

Reply:
wrong one ^^

Reply:
![image](https://user-images.githubusercontent.com/69105206/139539013-6e41d9f8-df30-4f52-9dd3-dedf9b2c9802.png)

Reply:
Well, you should commit that as that isn't on the repo. Perhaps run prettier over this directory?

Reply:
Will do

Reply:
One more thing, that `};` should be pushed to the left.